src/NzbDrone.Core/Extras/Subtitles/SubtitleFile.cs

```diff
@@ -1,10 +1,28 @@
-﻿using NzbDrone.Core.Extras.Files;
+﻿using System.Collections.Generic;
+using System.IO;
+using NzbDrone.Core.Extras.Files;
 using NzbDrone.Core.Languages;
 
 namespace NzbDrone.Core.Extras.Subtitles
 {
     public class SubtitleFile : ExtraFile
     {
         public Language Language { get; set; }
+
+        public string AggregateString => Language + Extension + LanguageTags;
+
+        public string LanguageTags { get; set; }
```

Comment:
Thinking about this a bit more, I think it'd make sense to make this an array and have the value stored in the DB as such (serialized as a JSON string), the logic to automatically serialize and deserialize already exists.

Reply:
Refactored in 24334c4.